Added a PCB for a piezo electric element

Supermagnum opened the pull request:
The problem with piezoelectric and contact mics is that they are not well matched to typical audio inputs. 
The reason why these devices often sound tinny is because the piezo sensor presents its signal through a series capacitance which is small, typically 15nF or less. When wired to a normal 50 kilohm line input this forms a 200Hz high-pass filter, which eliminates the bass. 
Because the piezo disc has a very high impedance (1 MΩ typical), it should be buffered to avoid possible impedance mismatch with an existing audio system. This circuit acts as a impedance buffer, and some amplification does not hurt as long as it is around 40dB. 
The best piezoelectric sensors for listening is based on Aluminum Nitride and molybdenum. PZT-5 also known as SensorTech BM500,Channel 5500,Morgan PZT5A1 is widely used.

NOAA technical memorandum NMFS 2008NOAA-TM-NMFS-SWFSC-417 also known as
A guide to constructing hydrophones and hydrophone arrays for monitoringmarine mammal vocalizations,
describes how to build a single crystal hydrophone.

If the piezo electric crystals are a suitable distance and 45° degrees apart from each other in a flat circular array with 8  piezoelectric sensors,
they should be compatible with Manyears or ODAS.

Approximate values for fresh water and seawater, respectively, at surface pressure are 1450 around 1500 m/s.
 The speed of sound in water increases with increasing pressure, temperature and salinity.

Also, read: https://github.com/Supermagnum/passiveSonar


It is made with Kicad.

Supermagnum committed:
added a pcb for a piezo electric element

Supermagnum committed:
Update README

Supermagnum committed:
Add files via upload

added picture of board dimentions

vrheaume commented:
Hello, 

I have looked at the PDF schematics and recommended not to pull this project for now. Has the circuit been tested and shown functional? 


Here are some comments: 
-I believe output of R1 should make its way to pin 3 of U1 for any amplification to be possible. 
-R1, R2 and R3 give strong attenuation to X1 output; is the gain of 10 from U1 sufficient? 
-I think J1-pin 3 should be grounded if differential configuration is not necessary. Otherwise, the floating input could contribute to noise. I recommend you ground it through a 0603 package, 0-ohm resistor. 


In addition to the transducer frontend, I think some validations and modifications _will_ have to be made to the 8Sounds and 16Sounds in order to work at the low frequencies. I suggest we discuss more via email. 

Vincent R., hardware designer for the 16SoundsUSB and other projects
V-P.Rheaume@USherbrooke.ca

Supermagnum commented:
> Hello,
> 
> I have looked at the PDF schematics and recommended not to pull this project for now. Has the circuit been tested and shown functional?
> 
> Here are some comments:
> -I believe output of R1 should make its way to pin 3 of U1 for any amplification to be possible.
> -R1, R2 and R3 give strong attenuation to X1 output; is the gain of 10 from U1 sufficient?
> -I think J1-pin 3 should be grounded if differential configuration is not necessary. Otherwise, the floating input could contribute to noise. I recommend you ground it through a 0603 package, 0-ohm resistor.
> 
> In addition to the transducer frontend, I think some validations and modifications _will_ have to be made to the 8Sounds and 16Sounds in order to work at the low frequencies. I suggest we discuss more via email.
> 
> Vincent R., hardware designer for the 16SoundsUSB and other projects
> [V-P.Rheaume@USherbrooke.ca](mailto:V-P.Rheaume@USherbrooke.ca)

I have written you a email, I hope it doesn't end up in a spam filter.
The circuit should work, as it is based on:
http://www.richardmudhar.com/blog/piezo-contact-microphone-hi-z-amplifier-low-noise-version/
Other op amps that is suitable is:
 OPA365,OPA1671,AD8613,TS922 and the one in the schematic.
There are no need to change the front-end, as it's intended for the human audio spectrum.

vrheaume commented:
I have no doubt that the original circuit works, however I believe a mistake happened when recopying it. The original has a connection between the input of the amplifier (clamping diodes) and the piezo element, which is missing from the left schematics. Seeing the design you used as inspiration, I understand now, that R1 is the biasing resistor and not in the sensing path. 
 
This inconsistency is what leads me to believe that the design in the pull request isn't quite ready yet. I promise I'll look into the design further when the schematic seems OK and you have your own testing to back it. 

Regarding the frequencies of interest -- I had (wrongly) assumed that your application would require sub-audible frequencies. With the information that the frequencies of interest do not span below the audible band, then I agree that the existing 8SoundsUSB and 16SoundsUSB acquisition boards can be used as-is. I highly suggest the 16SoundsUSB as it uses more recent chips. 

Your PDF schematics specifies the NE5534. I disagree that it is a good choice for 5V application. Looking at the datasheet, I'm pretty sure you'll run into an input common-mode voltage problem. I haven't looked at the rest of your list of opamps, but I suggest that you consider the MCP6V74 or similar -- I've had good success with them for the low-noise audio applications. 

I think your application is very nice. Let me know if I can help with the hardware.

Supermagnum committed:
Add files via upload

updated files

Supermagnum committed:
updated files

updated files.

Supermagnum committed:
Update README

Supermagnum commented:
Changed the opamp to a LME49724 , as requested it has differential outputs.

Supermagnum committed:
Add files via upload

updated files

Supermagnum committed:
Update README

Supermagnum committed:
Add files via upload

updated gerber files,

Supermagnum committed:
Add files via upload

Added a pair of zeners as a waveform clipper to protect the opamp inputs against damaging voltage spikes.
Also, updated pictures and other files.

Supermagnum committed:
Update README

Updated readme.

Supermagnum committed:
added 

Added LME49724 cooling footprint vias and CU backside.

Supermagnum committed:
Update README

Supermagnum committed:
Add files via upload

Updated gerber files

Supermagnum committed:
Add files via upload

updated files

Supermagnum committed:
Add files via upload

updated corrected schema

Supermagnum committed:
Add files via upload

updated files

Supermagnum commented:
Corrected diode connections.

Supermagnum committed:
Update README

Supermagnum committed:
Update README

Supermagnum committed:
updated gerbers

Supermagnum committed:
updated

Supermagnum committed:
updated files

Supermagnum committed:
Add files via upload

Supermagnum committed:
updated readme fule

Supermagnum committed:
Hopefully the final update

Supermagnum committed:
Final update, all should be OK now!

vrheaume commented:
Greetings again, 


I am reviewing the design you submitted. Here are questions, comments and recommendations of varying importance: 

1) Did you test the design? We can't pull something that hasn't been tested. 



Regarding the PCB: 

2) I see that you have a good low-impedance ground plane, however near U1 pin6 GND has to go through a narrow trace bottleneck. I think it would be better for this ground connections to add a via to the ground plane near U1 pin6. 

3) The ground trace near R9, R7, C5 partially overlaps the ground plane -- when zooming onto the Gerber file, it doesn't look pretty. I think the PCB manufacturer would flag that as a DfM violation. 

4) I don't like how some traces snake around, under and hug component pads -- depending on quality of soldermask alignment it looks like a risk for short-circuit. See J1 pin 1, C3, C6, R5, R6. Even though it might respect the design rules, it would be better, where possible, to route the traces a bit further from unrelated pads (doesn't cost more, and might improve manufacturing yield). 



Regarding the files and folders: 

5) Usually the philosophy is to put the "generated files" with a "release" tag attached to the git repository, instead of them being part of the version repository itself. I'm not entirely familiar with the workings of git so if it is unclear I could ask my colleague Dominic how this works when we get there. 

6) Aside from the gerber+drill files, there should be a BOM (bill of materials) so people can send this project to fabrication easily. The BOM should contain at least "manufacturer ID" and "manufacturer part number" parameter fields, for all components. The BOM should be generated automatically (I'm sure kicad does this!) so that we can re-generate it seamlessly if we ever need to make a revision. 



The circuit topology itself seems like it could work but I think that the comments above would improve the submission. I would also be more comfortable to have some feedback regarding your testing before I pull the design. 


Thank you for your submission, 

Vincent R., hardware designer for the 16SoundsUSB and other projects
V-P.Rheaume@USherbrooke.ca

Supermagnum commented:
I am in process of building and testing this, as I have issues soldering SMD components:
https://github.com/Supermagnum/piezo-balanced
